### ISSUE TITLE
_groupHover active in the whole viewport

### ISSUE THREAD

**Issue by secretlifeof** [chakra-ui/chakra-ui#372]
I am wandering how to use the _groupHover correctly. When using the composition below, hover is active in the whole viewport. What am I missing?

```
<PseudoBox pos="fixed" top="48%" role="group">
  <Flex align="center" cursor="pointer">
    <PseudoBox color="red" _groupHover={{color: 'brown'}}>
      <Icon name="zurueck" color="currentColor" px="1vw" w="6vw" h="auto" />
    </PseudoBox>
    <Text color="red" fontSize={["lg", "md", "md"]}>
      zurück
    </Text>
  </Flex>
</PseudoBox>
```

**Comment by trevorblades** [comment]
Are you using `@reach/router` or Gatsby (which uses `@reach/router`)? I'm seeing this as well since the router is adding wrapping the page in a div with `role="group"`. In any case, I suspect happening because you have an HTML element higher up in your page hierarchy with a `role="group"` attribute set.

I'm currently looking for a way to control the scope (??) of a group, so children of one group ignore the hover status of "grandparent" groups. I'll update this thread if I made any progress on this.

**Comment by trevorblades** [comment]
Update: https://github.com/gatsbyjs/gatsby/pull/21171 got merged today and this problem is now fixed as of Gatsby v2.19.15 🎉

**Comment by secretlifeof** [comment]
@trevorblades Yes I am using Gatsby. Thank you very much for feedback! Closing this Issue.
